db/models/index.js

```diff
@@ -1,43 +1,50 @@
-'use strict';
+'use strict'
 
-const fs = require('fs');
-const path = require('path');
-const Sequelize = require('sequelize');
-const process = require('process');
-const basename = path.basename(__filename);
-const env = process.env.NODE_ENV || 'development';
-const config = require(__dirname + '/../../config/config.js')[env];
-const db = {};
+const fs = require('fs')
+const path = require('path')
+const Sequelize = require('sequelize')
+const process = require('process')
+const basename = path.basename(__filename)
+const env = process.env.NODE_ENV || 'development'
+const config = require(__dirname + '/../../config/config.js')[env]
+const db = {}
 
-let sequelize;
+let sequelize
 if (config.use_env_variable) {
-  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+    sequelize = new Sequelize(process.env[config.use_env_variable], config)
 } else {
-  sequelize = new Sequelize(config.database, config.username, config.password, config);
+    sequelize = new Sequelize(
+        config.database,
+        config.username,
+        config.password,
+        config
+    )
 }
 
-fs
-  .readdirSync(__dirname)
-  .filter(file => {
-    return (
-      file.indexOf('.') !== 0 &&
-      file !== basename &&
-      file.slice(-3) === '.js' &&
-      file.indexOf('.test.js') === -1
-    );
-  })
-  .forEach(file => {
-    const model = require(path.join(__dirname, file))(sequelize, Sequelize.DataTypes);
-    db[model.name] = model;
-  });
+fs.readdirSync(__dirname)
+    .filter((file) => {
+        return (
+            file.indexOf('.') !== 0 &&
+            file !== basename &&
+            file.slice(-3) === '.js' &&
+            file.indexOf('.test.js') === -1
+        )
+    })
+    .forEach((file) => {
+        const model = require(path.join(__dirname, file))(
+            sequelize,
+            Sequelize.DataTypes
+        )
+        db[model.name] = model
+    })
 
-Object.keys(db).forEach(modelName => {
-  if (db[modelName].associate) {
-    db[modelName].associate(db);
-  }
-});
+Object.keys(db).forEach((modelName) => {
```

Comment:
using forEach is a bad practice in js, generally for...of option is better. You will not feel any difference in you code right now, but i recommend you to use for...of for the future

Reply:
I have read that forEach is slower than general for loop

Reply:
i will add a few reasons: 1) forEach does not support asynchrony, 2) you can't skip or break the loop inside forEach, 3) it is impure function so it can cause side effects.
There is only one case when you can use forEach confidently: you iterate array and work only with its elements without using external variables but such cases are not very common. So if you use for..of 10 times in your code and meet the appropriate situation for using forEach it is still better to use for.of to preserve code style